controllers/replication.storage/volumereplication_controller.go

```diff
@@ -125,6 +125,8 @@ func (r *VolumeReplicationReconciler) Reconcile(ctx context.Context, req ctrl.Re
 	}
 	// remove the prefix keys in volume replication class parameters
 	parameters := filterPrefixedParameters(replicationParameterPrefix, vrcObj.Spec.Parameters)
+	schedulingTime := parameters["schedulingInterval"]
+	scheduleTime, _ := time.ParseDuration(schedulingTime)
```

Comment:
This is optional and maybe empty ? 

do we need a default schedule time ? 

cc @Madhu-1

Reply:
Yes it can be empty and it's optional

Reply:
done